crypto/bio/bio_lib.c

```diff
@@ -62,7 +62,8 @@ static long bio_call_callback(BIO *b, int oper, const char *argp, size_t len,
     ret = b->callback(b, oper, argp, argi, argl, inret);
 
     if (ret >= 0 && (HAS_LEN_OPER(bareoper) || bareoper == BIO_CB_PUTS)) {
-        *processed = (size_t)ret;
+        if (processed != NULL)
+            *processed = (size_t)ret;
```

Comment:
I doubt this can be the right fix
because the return value is always 1
and not the number of written bytes